.env.example

```
@@ -1 +1,6 @@
 MODEL_DIR=/path/to/models
+
+# WandbLogger
+WANDB_API_KEY=your_api_key_here
+WANDB_ENTITY=your_username_or_team
+WANDB_PROJECT=your_project_name # optional, overrides constructor
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)